bundles/org.eclipse.orion.client.ui/web/orion/webui/littlelib.js

```diff
@@ -475,6 +475,27 @@ define(["orion/util"], function(util) {
 		return offsetParent;
 	}
 	
+	function getPathTo(element) {
+		if (!element) return "";
+		if (element.id !== "") {
+			return 'id("' + element.id +'")';
+		}
+		if (element === document.body) {
+			return element.tagName;
+		}
+		var ix= 0;
+		var siblings = element.parentNode.childNodes;
+		for (var i= 0; i < siblings.length; i++) {
+			var sibling = siblings[i];
+			if (sibling === element) {
+				return getPathTo(element.parentNode)+'/'+element.tagName+'['+(ix+1)+']';
+			}
+			if (sibling.nodeType === 1 && sibling.tagName === element.tagName) {

```

Comment:
Add comment ` // ELEMENT_NODE` to the end of this line for consistency with other refs to nodeType